packages/navigation/src/_internal/getCustomRenderFunction.tsx

```diff
@@ -1,21 +1,43 @@
-import React, { ReactNode } from "react";
-import { HeightSender } from "../header-height-context";
+import React, { PropsWithChildren, ReactNode } from "react";
 import { View } from "react-native";
+import { MadBaseOptions } from "./types";
 /**
  * Custom render function that displays a custom sub-header
  * @param originalRender the original render function, found in the descriptor
  * @returns new render function
  */
-export const getCustomRenderFunction = (
+export const getCustomRenderFunction = <T extends MadBaseOptions>(
     originalRender: () => ReactNode,
+    options: T,
     CustomSubHeader?: () => ReactNode,
```

Comment:
Should `options` and `CustomSubHeader` be added to JSDoc?

Reply:
Fixed